source/configure/environment-configuration-settings.rst

```diff
@@ -3643,6 +3644,95 @@ Allow untrusted internal connections
 |   but not ``10.0.16.20``.                                                                                                                     |
 +-----------------------------------------------+-----------------------------------------------------------------------------------------------+
 
+Mobile security
+---------------
+
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
+
+From Mattermost v10.7 and mobile app v2.27, you can configure biometric authentication, prevent Mattermost use on jailbroken or rooted devices, and can block screen captures without relying on an EMM Provider. Configure these options by going to **System Console > Environment > Mobile Security**, or by editing the ``config.json`` file as described in the following tables. Changes to configuration settings in this section require a server restart before taking effect.
```

Comment:
Is it true that a server restart is required for these configs?

Reply:
Elias knows for sure, but almost sure this was meant in the case you manually update the `config.json` file.

Reply:
A server restart is not needed IF you reload the configuration, once the settings are available in the admin console it will be applied as you click on save